tests/integration-tests/conftest.py

```diff
@@ -156,6 +156,9 @@ def pytest_addoption(parser):
     parser.addoption(
         "--no-delete", action="store_true", default=False, help="Don't delete stacks after tests are complete."
     )
+    parser.addoption(
+        "--retain-ad-stack", action="store_true", default=False, help="Retain AD stack and corresponding VPC stack."
```

Comment:
Let's include this new option in the README where all test runner options are documented.